lib/awful/placement.lua

```diff
@@ -827,17 +834,19 @@ end
 --- Place the client so no part of it will be outside the screen (workarea).
 --@DOC_awful_placement_no_offscreen_EXAMPLE@
 -- @client c The client.
--- @tparam[opt=client's screen] integer screen The screen.
+-- @tparam[opt={}] table args The arguments
+-- @tparam[opt=client's screen] integer args.screen The screen.
 -- @treturn table The new client geometry.
-function placement.no_offscreen(c, screen)
+function placement.no_offscreen(c, args)
     --HACK necessary for composition to work. The API will be changed soon
```

Comment:
This comment is now wrong / outdated, right?

Reply:
updated the comment and added deprecation warning